doc/collide_modify_swpm.txt

```diff
@@ -0,0 +1,130 @@
+"SPARTA WWW Site"_sws - "SPARTA Documentation"_sd - "SPARTA Commands"_sc :c
+
+:link(sws,http://sparta.sandia.gov)
+:link(sd,Manual.html)
+:link(sc,Section_commands.html#comm)
+
+:line
+
+collide_modify swpm command :h3
+
+[Syntax:]
+
+collide_modify keyword values ...  :pre
+
+one or more keyword/value pairs may be listed :ulb,l
+keywords = {swpm} or {reduce}
+  {swpm} values = choice Nsplit gamma
+    choice = {yes} or {no} to allow particle weights to stochastistically change
+    Nsplit = maximum number of particles in a cell to split particles
+    gamma = weight transfer functino which is a value > 0
+  {reduce} values = reduction_choice Nreduce group_choice Ngroup
+    reduction_choice = {ENERGY} or {HEAT} or {STRESS} = how to reduce particles
```

Comment:
We typically don't have input deck options as all caps, e.g. ENERGY, HEAT, STRESS, can you please make these lowercase instead?